css/css-flexbox/flexbox_justifycontent-right-001-ref.html

```diff
@@ -9,11 +9,9 @@
   width: 40em;
   display: flex;
   flex-direction: row;
-  justify-content: right;
```

Comment:
I'm not sure I'm getting this properly. So despite you set `justify-content: right`, the things are on the left of the flex container. Is that the expected behavior?